theme/layouts/default.ejs

```diff
@@ -6,10 +6,11 @@
   <body>
     <header>
       <div>
```

Comment:
Suggestion:  outer `<div>` doesn't seem to be necessary here, let's remove it, to simplify our code unless it's here for a reason.

Reply:
OK